src/main/java/domain/BlackJackGame.java

```diff
@@ -65,6 +72,11 @@ public Map<GameResult, Integer> calculateDealerWinningCount() {
         return dealerWinningCount;
     }
 
+    public Money calculateDealerRevenue() {
+        Map<Player, Money> results = calculateRevenue();
+        return results.values().stream().reduce(Money::plus).map(money -> money.times(-1)).orElse(Money.of(0));
+    }
```

Comment:
사소하지만 개행을 해주면 메서드체인 호출에 대한 가독성이 증가해요!

```suggestion
        return results.values()
                .stream()
                .reduce(Money::plus)
                .map(money -> money.times(-1))
                .orElse(Money.of(0));
```